JTCalendar/JTCircleView.m

```diff
@@ -19,7 +19,7 @@ - (instancetype)init
     }
     
     self.backgroundColor = [UIColor clearColor];
-    self.color = [UIColor whiteColor];
+    self.color = [UIColor clearColor];
```

Comment:
I made this modification because when `useDayCircleColorForSelected` was set to `NO`, the selected day had the circle color set to white.